frontend/model/notifications/templates.js

```diff
@@ -142,14 +132,17 @@ export default ({
     }
   },
   NEW_PROPOSAL (data: { groupID: string, creatorID: string, subtype: NewProposalType }) {
-    const name = strong(userDisplayNameFromID(data.creatorID))
+    const args = {
+      name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.creatorID}`,
```

Comment:
Actually, this is my intentional update to fix the following confusing problem.
```
The group has a new member. Say hi to alexjin!
Alex Jin proposed to add a member to the group. Vote now!
Alex created a proposal. Vote now!
SilverIT  proposed to change the group voting system. Vote now!
```
Just use identityContractID here, and we will swap it for displayName or username in `NotificationList.vue` using `swapMentionIDForDisplayname` function.

Reply:
And further more, I think if we use identityContractID there, we could place profileCard just like how we did in Chat messages.
That could be useful because there could be several members that have same display name. 😁